components/popover/index.css

```diff
@@ -10,136 +10,362 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+}
+
+/* windows high contrast mode */
+@media (forced-colors: active) {
+  .spectrum-Popover {
+    --highcontrast-popover-border-color: CanvasText;
+  }
 }
 
 .spectrum-Popover {
   @inherit: %spectrum-overlay;
 
-  /* Be a flexbox to allow a full sized content area that scrolls */
-  display: inline-flex;
-  flex-direction: column;
   box-sizing: border-box;
+  position: absolute;
 
-  min-width: var(--spectrum-popover-min-width);
-  min-height: var(--spectrum-popover-min-height);
+  outline: none; /* Hide focus outline */
 
-  position: absolute;
+  padding: var(--mod-popover-content-area-spacing-vertical, var(--spectrum-popover-content-area-spacing-vertical)) 0;
+
+  display: inline-flex;
+  flex-direction: column;
 
   border-style: solid;
-  border-width: var(--spectrum-popover-border-size);
-  border-radius: var(--spectrum-popover-border-radius);
+  border-radius: var(--mod-popover-corner-radius, var(--spectrum-popover-corner-radius));
+  border-color: var(--highcontrast-popover-border-color, var(--mod-popover-border-color, var(--spectrum-popover-border-color)));
+  border-width: var(--mod-popover-border-width, var(--spectrum-popover-border-width));
 
-  outline: none; /* Hide focus outline */
-  box-sizing: border-box;
+  background-color: var(--mod-popover-background-color, var(--spectrum-popover-background-color));
+  filter: drop-shadow(var(--mod-popover-shadow-horizontal, var(--spectrum-popover-shadow-horizontal)) var(--mod-popover-shadow-vertical, var(--spectrum-popover-shadow-vertical)) var(--mod-popover-shadow-blur, var(--spectrum-popover-shadow-blur)) var(--mod-popover-shadow-color, var(--spectrum-popover-shadow-color)));
 
+  /* default opens and animates upward */
   &.is-open {
     @inherit: %spectrum-overlay--open;
+    @inherit: %spectrum-overlay--top--open;
+  }
+
+  /* has tip triangle */
+  &.spectrum-Popover--withTip {
+    .spectrum-Popover-tip {
+      /* triangle polygon */
+      .spectrum-Popover-tip-triangle {
+        stroke-linecap: square;
+        stroke-linejoin: miter;
+        fill: var(--mod-popover-background-color, var(--spectrum-popover-background-color));
+        stroke: var(--mod-popover-border-color, var(--spectrum-popover-border-color));
+        stroke-width: var(--mod-popover-border-width, var(--spectrum-popover-border-width));
+      }
+    }
   }
 }
 
-.spectrum-Popover-tip {
-  position: absolute;
-  --spectrum-popover-tip-size: var(--spectrum-popover-tip-width);
-  --spectrum-popover-tip-borderWidth: var(--spectrum-popover-border-size);
-  /* https://stackoverflow.com/questions/44170229/how-to-prevent-half-pixel-svg-shift-on-high-pixel-ratio-devices-retina */
-  -webkit-transform: translate(0, 0);
+/* position naming - first position term is popover position, second term is source position */
+/* example: --top-left = popover is to top and source is to left */
```

Comment:
This is a helpful comment that I think would be useful to have in the component Usage notes if there is time to include it.

Reply:
Good idea. Updated in new commit.